docs/install/containerd/containerd-install.md

```diff
@@ -0,0 +1,99 @@
+# Install Kata Containers with containerd
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && { echo >&2 "ERROR: Please remove existing Kata Containers installation"; exit 1; }
+   ```
+
+1. Install dependencies
+
+   ```bash
+   $ source /etc/os-release || source /usr/lib/os-release
+   $ packages='containerd curl git jq'
+   $ case "$ID" in
+   $   ubuntu) sudo apt-get -y install $packages ;;
```

Comment:
debian should work too, shouldn't it?

Reply:
In theory, but see https://github.com/kata-containers/kata-containers/pull/739#issuecomment-693443741.

Reply:
These packages including `containerd` are available in Fedora as well.. Can we include `fedora` case as well ? 
cc @fidencio

Reply:
@bpradipt - I'd like to, but see https://github.com/kata-containers/kata-containers/pull/739#issuecomment-693443741.

Reply:
Ah I see. Yeah that's a problem. Can we add a comment that Fedora needs to be booted with cgroup v1 ? (`systemd.unified_cgroup_hierarchy=0`)

Reply:
Yes. It's a shame as it means we cannot "test" the document in the CI on Fedora 32. I plan to change the doc to error if cgroups v2 is detected, along with a helpful message.

I've discussed the general approach with @chavafg and as a result of that currently reworking this doc to install containerd from the containerd binary packages rather than distro packaged (due to all the current problems related to the containerd distro packages). That will allow the doc to work for all distros (except F32 - see above). I'll also add a warning since the user will need to consider updating Kata and containerd as and when new releases are available.